draft-ietf-quic-transport.md

```diff
@@ -3381,51 +3382,39 @@ Header Form:
 
 : The most significant bit (0x80) of byte 0 is set to 0 for the short header.
 
-Key Phase Bit:
-
-: The second bit (0x40) of byte 0 indicates the key phase, which allows a
-  recipient of a packet to identify the packet protection keys that are used to
-  protect the packet.  See {{QUIC-TLS}} for details.
-
-\[\[Editor's Note: this section should be removed and the bit definitions
-changed before this draft goes to the IESG.]]
+Fixed Bit:
 
-Third Bit:
+: The next bit (0x40) of byte 0 is set to 1.  Packets containing a zero value
```

Comment:
Just wanted to note that forcing bit 0x40 to 1 is critical to Google's demultiplexing. We were using 0x08 (the "Google QUIC Demultiplexing Bit") but since 0x08 is now encrypted we'll be using this one. So please keep this at 1, at least for short headers. Thanks!

Reply:
Agreed: keeping 0x40 will mean that servers will be able to support pre-Q044 clients as well.

Reply:
@DavidSchinazi, we're not doing this for you, but I'm glad you're happy.

Reply:
I realize that, but I just wanted to let you know it was important to us (i.e. please let us know if you plan to change this)

Reply:
I agree with @DavidSchinazi.  This is important to us (@litespeedtech) as well.  Please give a heads up should the plans for bit 0x40 change.